docs/ros/config/yaml.mdx

```diff
@@ -6,3 +6,340 @@ toc_min_heading_level: 2
+
+At this moment, the supported robot platforms are:
+1. [A200 Husky](/docs/robots/outdoor_robots/husky/user_manual_husky/#introduction): `a200-0000`
+2. [J100 Jackal](/docs/robots/outdoor_robots/jackal/user_manual_jackal/#introduction): `j100-0000`
+
+Every robot platform has specific decorations, i.e. platform specific parts, that are selected based on the serial number passed. Therefore, it is required that a serial number is specified in the `robot.yaml`.
+
+### Sample
+In our sample, we use a **J100 Jackal** and therefore have set our serial number to:
+```yaml
+serial_number: "j100-0000"
+```
+
+
+## System
+Proper networking setup is crucial in setting up the ROS2 middleware and ensure other onboard computers communicate reliably.
+
+
+### Hosts
+The **hosts** section serves as a way to match IP addresses to hostnames. By default, Clearpath robots use the serial number as the hostname and have a default IP of `192.168.131.1`.
+
+- the **self** entry refers to what hostname is this configuration file on.
+- the **platform** entry refers to the hostname and IP of the robot platform's main computer.
+- the **onboard** entry is used to define the hostname and IP of other computers on the robot.
+- the **remote** entry is used to define the hostname and IP of computers in the system that are not on the robot, such as a user's PC.
+
+### ROS2 Environment
+The **ros2** sections is necessary to setup the ROS2 middleware.
+- **username** must match the username of the user that is to run all ROS nodes.
+- **namespace** specified will be appended as a prefix to all sensor topics to prevent topic cloberring when multiple robots are on the same network and domain ID.
+- **domain_id** specifies the ROS2 domain ID to use.
+- **rmw_implementation** specifies the ROS2 middleware to use. **Currently, it only supports `rmw_fastrtps_cpp`.**
+
+
+### Sample
+In our sample, we have a *J100 Jackal* platform whose primary computer has hostname: `cpr-j100-0000` and IP: `192.168.131.1`.
+
+And, note that this configuration YAML is meant to be on that primary computer, hence `self: cpr-j100-0000`.
+
+By default, all Clearpath robots use username `administrator` and the robot's namespace matches the `serial_number`.
+
+<details><summary>Sample J100 System Section</summary>
+<p>
+
+```yaml
+system:
+  hosts:
+    self: "cpr-j100-0000"
+    platform:
+      cpr-j100-0000: "192.168.131.1"
+    onboard: {}
+    remote: {}
+  ros2:
+    username: "administrator"
+    namespace: "j100_0000"
+    domain_id: 0
+    rmw_implementation: "rmw_fastrtps_cpp"
+```
+
+</p>
+</details>
+
+## Platform
+Every robot platform has unique structures just as versatile sensor mounting solutions, wireless charging receivers, and waterproofing enclosures; we refer to these as decorations.
```

Comment:
Decorations seems misaligned with other Clearpath branding terms. Brackets seems more aligned with  existing Clearpath webpages. Consider reviewing this with someone from Marketing.

Reply:
You can also use OTTO's glossary of Product / Branding terms:
https://wiki.clearpathrobotics.com/pages/viewpage.action?pageId=106572126

Reply:
Taking a look at the glossary, the closest entry to what a **decoration** would be is an **attachment**. 

| attachment | Additional hardware products that are installed on robots to enhance their capabilities. For example lifts, conveyors, and arms. | DO NOT USE: Accessory, Appliance
-- | -- | --

**Decorations** was used in the ROS 1 Husky code to refer to *attachments* that were unique to the Husky, such as bumpers, top plates, and sensor archs. 

The **attachment**, as is defined by OTTO, applies to all entities in the config, i.e. decorations, accessories (probably needs changing), mounts, and sensors. 

I think that the word **decorations** has connotations of an ornamental, non-functional item; and, it does not fit the concept of a charging bumper, versatile top plate, or sensor arch.  Therefore, it will seem out of place to anyone not familiar with the Husky codebase.